### PR TITLE
First pass at no crop trampling

### LINKED ISSUE #48: SAG Mill missing recipes for many mod ingots (Immersive Engineering, Thermal Foundation, etc.)
## Issue Description

The Ender IO SAG Mill is missing grinding recipes for ingots from various mods, preventing players from converting them to dust for processing consistency.

## Affected Items
- Immersive Engineering nickel ingots
- Thermal Foundation nickel ingots
- Other modded metal ingots from various mods

## Expected Behavior
SAG Mill should be able to grind most metal ingots into their corresponding dust forms, similar to how it works with vanilla and common modded ingots.

## Impact
This creates inconsistency in processing chains and forces players to use alternative methods (like pulverizers) that may not be available early game or may have different output ratios.

## Suggested Solution
Add missing SAG Mill recipes for common modded metal ingots, particularly focusing on metals that have dust equivalents in the modpack.

### PR DESCRIPTION
**Pull request by cwage**
## Summary
Adds comprehensive farmland trampling protection using Trample No More mod with custom configuration and OpenLoader datapacks.

## Changes Made
- **Added Trample No More mod** and Bookshelf dependency to setup.sh
- **Configured Trample No More** to enable all protection mechanisms (featherFalling, taggedItems, taggedEntities, creativeMode, barefoot)
- **Created OpenLoader datapack** to tag players (`minecraft:player`) for complete trampling prevention
- **Created OpenLoader datapack** to tag common boots as "soft":
  - `minecraft:leather_boots`
  - `mekanism:free_runners` 
  - `enderio:dark_steel_boots`
- **Updated CLAUDE.md** to emphasize setup.sh-only workflow and never use packwiz commands directly

## How It Works
This solution prevents farmland trampling through multiple layered mechanisms:

1. **Entity Tag Protection**: Players are tagged to never trample farmland regardless of what they're wearing
2. **Soft Boots Protection**: Common early/mid-game boots are tagged as "soft" to prevent trampling
3. **Barefoot Protection**: Enabled for any edge cases not covered by the above
4. **All Protections Active**: All Trample No More features enabled for maximum coverage

## Test Results
✅ Farmland trampling is now completely prevented in all tested scenarios

Closes #48

### CODE REVIEW

File: cwagecraft/pack.toml [hunk 1]
```diff
@@ -6,11 +6,11 @@ pack-format = "packwiz:1.1.0"
...
-hash = "9672283f4c2e2c2035938d13ceef5ad8132db378a96cdba945f8f35d70e38e59"
+hash = "75614ff33f9fca274bc4128cac3d219ebf7e6aa494ce61a3c91a429056d7cf77"
 
 [versions]
 forge = "47.3.22"
 minecraft = "1.20.1"
 
 [options]
-acceptable-game-versions = ["1.20", "1.20.1"]
+acceptable-game-versions = ["list"]
```
[the remark this answers is not in the document]
Comment (cwage) [reply cwagecraft/pack.toml:16]: ✅ Fixed - Changed acceptable-game-versions from ["list"] to ["1.20", "1.20.1"] to properly specify supported Minecraft versions.
